buildSrc/src/main/groovy/io.deephaven.java-toolchain-conventions.gradle

```diff
@@ -25,10 +25,12 @@ if (testLanguageLevel < 11) {
   throw new IllegalArgumentException("testLanguageLevel must be greater than or equal to 11")
 }
 if (runtimeVersion < languageLevel) {
-  throw new IllegalArgumentException("runtimeVersion must be greater than or equal to languageLevel")
+  project.logger.lifecycle("runtimeVersion was set to ${runtimeVersion}, updating to ${languageLevel} to match languageLevel")
```

Comment:
I wonder if we should set 17 as the default, and then explicitly override the parts that we want to be 11? (And then I don't think we would need this?)

Reply:
Got it. Arguably, setting `-PtestRuntimeVersion` for our nightly testing is a "heavy hammer" that has worked well enough up to this point... ideally, we wouldn't need to do `server-jetty` checks (or other project checks) when this precondition is not true. While we could try to be finer with our configuration, I see the value in simply loosening the check here...

That said, I _think_ this is only possible in the `-Pjetty11` case, right? Ideally, this logic would revert back to the strict check once jetty-11 is deleted. I think we can retain the original behavior of throwing an exception normally, and only auto-upgrade the versions when `-Pjetty11` is set? (This will also be a nice marker to delete the workaround once jetty-11 is deleted.)

Reply:
Well, if we'd had this previously, we could have tested the java client under Java 8, which could have been beneficial.

As it stands, Jetty 11 isn't going away for at least a year, quite probably a whole lot longer.